GenderPayGap.WebUI/Views/AccountCreation/AlreadyCreatedAnAccountQuestion.cshtml

```diff
@@ -76,15 +76,18 @@
                 })
 
                 <div class="govuk-inset-text">
-                    You will need to create your own user account even if someone else has previously reported your
-                    organisation's gender pay gap.
+                    You will need to create your own user account even if someone else has
+                    previously reported your employer’s gender pay gap.
                 </div>
 
                 @(Html.GovUkButton(new ButtonViewModel
                 {
                     Text = "Continue"
                 }))
+                @{ var reportingUrl = Url.Action("ManageOrganisationsGet", "ManageOrganisations"); }
```

Comment:
Why do we want the user to be redirected to the manage organisations page if he doesn't have an account?

Reply:
I've just copied the same behaviour as the rest of the places where a similar pattern is used. 

If they don't have an account/aren't signed in it'll just redirect to the login page, if they are signed in and ended up here somehow, then that Manage Organisations page is the same place that login takes you.

For live, this isn't really relevant anyway since Global.StartUrl isn't null, we'll only see this behaviour on the other environments

Reply:
Exactly - the ManageOrganisations page checks whether the user is signed in, and if not they're auto-directed back to the sign in page. This actually happens in a number of places on the service (and we've got helper methods for it), but ManageOrganisations generally feels like a sensible 'home page' style place for logged in users, which it a good place to check if the user needs to sign in.